.node-version

```diff
@@ -0,0 +1 @@
+5.2
```

Comment:
this file and the test config file should list the same version (so either this should be `5.2.0` or the test should be `5.2`)